doc/mgr/nfs.rst

```diff
@@ -24,12 +24,14 @@ see :ref:`nfs-ganesha-config`.
 NFS Cluster management
 ======================
 
+.. _nfs-module-cluster-create:
+
 Create NFS Ganesha Cluster
 --------------------------
 
 .. code:: bash
 
-    $ ceph nfs cluster create <cluster_id> [<placement>] [--port <port>] [--ingress --virtual-ip <ip>]
+    $ nfs cluster create <cluster_id> [<placement>] [--ingress] [--virtual_ip <value>] [--ingress-mode {default|keepalive-only}] [--port <int>]
```

Comment:
Do we have any place in the docs where `default` behaviour is explained? otherwise I think it'd be good to add some note to clarify what's the default behaviour.

Reply:
sure, default here is just the default ingress, i.e. haproxy and keepalived. I can maybe edit the doc a bit to point that out somewhere.

Reply:
added another paragraph to the doc here explaining that default mode is the same as not passing the flag at all